pipeline/sign/sign.go

```diff
@@ -0,0 +1,102 @@
+package sign
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/goreleaser/goreleaser/context"
+	"github.com/goreleaser/goreleaser/pipeline"
+)
+
+type Pipe struct{}
+
+func (Pipe) String() string {
+	return "signing artifacts"
+}
+
+
+func (Pipe) Run(ctx *context.Context) error {
+	// set defaults
```

Comment:
defaults can be set by implementing the `pipeline.Defaulter` interface and adding it to the `pipeline/defaults` pipe. :)